probe_src/libprobe/src/util.c

```diff
@@ -1,3 +1,20 @@
+#include <stdio.h>  
+#include <stdlib.h>  
```

Comment:
Is there another CI tool we can bring in to eliminate _unneeded_ includes? If we delete a function in `util.c` it may cause the include to be unneeded.

Reply:
will check this